src/common/transport/transport-client.js

```diff
@@ -0,0 +1,80 @@
+import { TransportClientError } from '../errors';
+import rp from 'request-promise';
```

Comment:
I believe we could enable CORS. I don't know if we have specifically enabled it yet